api/src/test/java/org/pmiops/workbench/db/dao/InstitutionDaoTest.java

```diff
@@ -0,0 +1,71 @@
+package org.pmiops.workbench.db.dao;
+
+import static com.google.common.truth.Truth.assertThat;
+import static com.google.common.truth.Truth8.assertThat;
+
+import java.util.Optional;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.pmiops.workbench.db.model.DbInstitution;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@DataJpaTest
+public class InstitutionDaoTest {
+  @Autowired InstitutionDao institutionDao;
+
+  @Test
+  public void testDao() {
```

Comment:
Would it make sense to have separate tests for `save`, `findOneByShortName `, `findAll` and `delete`? Instead of having everything in one test. Also, does it makes sense to add a method for data setup that runs between each test? Something like:
`@Before`
 ` public void setUp() {`
       `final DbInstitution testInst = new DbInstitution("Broad", "The Broad Institute");`
      ` institutionDao.save(testInst);`
  `}`